tests/datasets/test_transaction_processor.py

```diff
@@ -304,6 +306,9 @@ def test_missing_trace_context(self) -> None:
         assert processor.process_message(payload, meta) is None
 
     def test_base_process(self) -> None:
+        old_skip_context = settings.TRANSACT_SKIP_CONTEXT_STORE
```

Comment:
is this test actually testing the skipping functionality?

Reply:
yes it does because of this line 
https://github.com/getsentry/snuba/pull/2009/files#diff-e3662d872d836735db5314fa78ade9a984552005a23495b472c0157815b26f9eR127

The event to compare contains an `experiments` context that is configured to be skipped and the expected results does not.